client/rust/axis/capture-jpeg-snapshot.c

```diff
@@ -0,0 +1,31 @@
+#define _GNU_SOURCE
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <errno.h>
+#include <signal.h>
+#include <sys/prctl.h>
+
+#include <capture.h>
+
+// capture-jpeg-snapshot: captures a jpeg snapshot from an Axis camera via libcapture.so API and writes it to stdout
+// this is necessary since the main Rust executable is static and can't currently link to libcapture.so
```

Comment:
I suppose we could, especially since the fix should be rather trivial, however we would not be able to use the result since there'll be large proportion of cameras in the wild whose firmware won't be upgraded